clang/lib/Sema/SemaDecl.cpp

```diff
@@ -2781,6 +2781,12 @@ static bool mergeDeclAttribute(Sema &S, NamedDecl *D,
     NewAttr = S.MergeSYCLUsesAspectsAttr(D, *A);
   else if (const auto *A = dyn_cast<SYCLIntelPipeIOAttr>(Attr))
     NewAttr = S.MergeSYCLIntelPipeIOAttr(D, *A);
+// else if (const auto *A = dyn_cast<SYCLDetailDeviceGlobalAttr>(Attr))
+//    NewAttr = S.MergeSYCLDetailDeviceGlobalAttr(D, *A);
```

Comment:
I'm not sure that we need these merge functions, since the attributes for `device_global` don't have any values. In fact, we probably need to stay with a simple handler for them. But it would be better to clarify this with someone who has more experience with attributes.

Reply:
We definitely don't. I should've simply removed those lines instead of commenting them out.